parts/linux/cloud-init/artifacts/sync-tunnel-logs.sh

```diff
@@ -39,7 +39,7 @@ while true; do
 done &
 
 # Manually sync all matching logs once
-for TUNNEL_LOG_FILE in $(compgen -G "$SRC/@(aks-link|azure-cns|cilium|konnectivity|tunnelfront)-*_kube-system_*.log"); do
+for TUNNEL_LOG_FILE in $(compgen -G "$SRC/*_kube-system_*.log"); do
```

Comment:
I suppose TUNNEL_LOG_FILE is no longer accurate ;)

Reply:
TUNNEL_LOG_FILE -> KUBE_SYSTEM_LOG_FILE, or maybe just LOG_FILE would work

Reply:
renamed to CONTAINER_LOG_FILE.